lib/login/accountCreation.dart

```diff
@@ -0,0 +1,199 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:off_top_mobile/models/User.dart';
+
+import 'package:http/http.dart' as http;
+import 'package:off_top_mobile/login/login.dart';
+
+class SignUp extends StatefulWidget {
+  const SignUp({@required this.email});
+  final String email;
+  @override
+  _SignUpState createState() => _SignUpState();
+}
+
+class _SignUpState extends State<SignUp> {
+  final TextEditingController firstNameController = TextEditingController();
+  final TextEditingController lastNameController = TextEditingController();
+  final TextEditingController cityController = TextEditingController();
+  final TextEditingController ageController = TextEditingController();
+  final TextEditingController genderController = TextEditingController();
+  final TextEditingController professionalController = TextEditingController();
+  final TextEditingController usernameController = TextEditingController();
+
+  bool visible = false;
+  Future<void> makePostRequest() async {
+    setState(() {
+      visible = true;
+    });
+    debugPrint('In postrequest');
+
+    final String email = widget.email;
+    final String password = 'HoldTheDoor';
+    final String deletedAt = 'null';
+    final String firstName = firstNameController.text;
+    final String lastName = lastNameController.text;
+    final String city = cityController.text;
+    final String age = ageController.text;
+    final String gender = genderController.text;
+    final String professional = professionalController.text;
+    final String username = usernameController.text;
+    final DateTime now = DateTime.now();
+    final String formatDate =
+        '${now.month.toString()}/${now.day.toString()}/${now.year.toString()}';
```

Comment:
wrap this into a method

Reply:
Putting these variables into a method will break Object parameters for User.dart.